test/test_topology_platforms_shell.py

```diff
@@ -215,7 +215,8 @@ def test_is_connected(spawn, shell):
     Test that is_connected returns correctly.
     """
 
-    assert not shell.is_connected()
```

Comment:
Ok, I agree with this, a shell that is just created does not have a disconnected default connection, instead it has no default connection.